checker/src/main/java/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -157,11 +180,38 @@
                     "android.annotation.Nullable",
                     // https://android.googlesource.com/platform/frameworks/support/+/master/annotations/src/main/java/android/support/annotation/Nullable.java
                     "android.support.annotation.Nullable",
+                    // TODO: found references, but no canonical declaration, maybe generated?
```

Comment:
Here is an old declaration:

https://android.googlesource.com/platform/tools/metalava/+/9ad32fadc5a22e1357c82b447e33ec7fecdcd8c1/stub-annotations/src/main/java/android/support/annotation/RecentlyNullable.java

I do see indication that it's not normally used in source code but rather inserted later:

> There are some annotations here which are not in the support library, such as @RecentlyNullable and @RecentlyNonNull. These are used _only_ in the stubs to automatically mark code as recently annotated with null/non-null. We do _not_ want these annotations in the source code; the recent-ness is computed at build time and injected into the stubs in place of the normal null annotations.

https://android.googlesource.com/platform/tools/metalava/+/96c2d484894d61586dee01454b1b89112edfde81/stub-annotations/README.md

That class changed packages as part of the move from android.support to androidx:

https://android.googlesource.com/platform/tools/metalava/+/5bda8555401bc13ef9009ad174274768302c7d9d

That may well mean that the old name is obsolete nowadays. It's just hard to be sure if it's being inserted automatically :\